server/src/main/java/io/spine/server/entity/Migration.java

```diff
@@ -75,8 +76,9 @@
     private @Nullable Operation<I, S, E> currentOperation;
 
     /**
-     * Applies the migration to a given entity, starting a new migration
-     * {@linkplain Operation operation}.
+     * Applies the migration {@linkplain Operation operation} to a given entity.
+     *
+     * @see Operation Migration.Operation for details
```

Comment:
This `see` section looks redundant. You have the very same link as for `operation`. I'd better remove it.